releasing/plugin-commands-publishing/src/pack.ts

```diff
@@ -111,6 +112,8 @@ export async function handler (
       [path.join(dir, 'package.json')]: publishManifest as Record<string, unknown>,
     },
   })
+  // display order by name
+  const packFiles = files.sort().join('\n')
```

Comment:
Let’s make sort more predictable:
```suggestion
  const packFiles = files.sort((a, b) => a.localeCompare(b, 'en')).join('\n')
```

Reply:
Also, this line of code is not needed at this point. Only at the end of the function